servers.json

```diff
@@ -1566,5 +1566,13 @@
 	    "voidedcraft.gamename.net",
 	    "voidedcraft.mcalias.com"
         ]
+    },
+    {
+        "id": "voltyum",
+        "name": "Voltyum Network",
+        "addresses": [
+            "play.voltyum.net",
```

Comment:
Look over the checklist again, mention me once you've fixed this

Reply:
@GrowlyX I've just fixed it, sorry :)